### ISSUE TITLE
files in scripts directory importing ga4gh

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#1083]
Some files in the `scripts` directory are importing the `ga4gh` package, which they do not have access to (if it is not installed):
```
$ python scripts/prepare_compliance_data.py -o ga4gh-example-data
Traceback (most recent call last):
  File "scripts/prepare_compliance_data.py", line 18, in <module>
    import generate_gff3_db
  File "/Users/dannyc/ga4gh/server/scripts/generate_gff3_db.py", line 20, in <module>
    import ga4gh.gff3Parser as gff3
ImportError: No module named ga4gh.gff3Parser
```
We need to either remove references to the `ga4gh` package within the `scripts` directory or figure out some import magic to ensure that the scripts are using the development ga4gh rather than the installed ga4gh.

**Comment by dcolligan** [comment]
Right now the only offenders are `generate_gff3_db.py` and `demo_example.py`

**Comment by dcolligan** [comment]
Fixed in #1084
